src/vscode-dts/vscode.proposed.mcpConfigurationProvider.d.ts

```diff
@@ -38,7 +38,20 @@ declare module 'vscode' {
 		 */
 		env: Record<string, string | number | null>;
 
-		constructor(label: string, command: string, args?: string[], env?: Record<string, string | number | null>);
+		/**
+		 * Optional version identification for the server. If this changes, the
+		 * editor will indicate that tools have changed and prompt to refresh them.
```

Comment:
Should this work by vscode looking at the toolset and comparing it, rather than the extension/mcp server declaring the version changed?

Reply:
We only get the toolset after the MCP is actually _started_, which we don't do eagerly. This serves as an indicator that the toolset may have changed and that we need to start it to check for any changes.